_overviews/toolkit/web-server-query-parameters.md

```diff
@@ -0,0 +1,77 @@
+---
+title: How to handle query parameters?
+type: section
+description: Handling query parameters with Cask
+num: 33
+previous-page: web-server-dynamic
+next-page: web-server-input
+---
+
+{% include markdown.html path="_markdown/install-cask.md" %}
+
+You can read the query parameters by adding to an endpoint method arguments that don't have corresponding path segments
```

Comment:
Consider starting with a sentence that reminds/tells the reader what "query parameters" even are.